spec/policies/user_policy_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+RSpec.describe UserPolicy do
```

Comment:
I'm thinking if there's an easy way for us to re-use the policy matcher we have in the Service app. Because that one will automatically test any new permissions as well and check they're not allowed, unless white listed.
Because when somebody does add a new policy rule, it may get missed from the `is_expected.not_to` list of assertions.